addons/test_mail/tests/test_performance.py

```diff
@@ -313,7 +313,7 @@ def test_mail_composer(self):
                 'partner_ids': [(4, customer_id)],
             })
 
-        with self.assertQueryCount(__system__=32, employee=38):
+        with self.assertQueryCount(__system__=33, employee=39):
```

Comment:
Do we still need to bump the query counters now that we check for the presence of the rating mixin before doing the query?

Reply:
according to the runbot, yes it bumped...